apps/web/examples/phoneInput/phoneInput.root.tsx

```diff
@@ -0,0 +1,52 @@
+import { PhoneInput } from "flowbite-react";
+import type { CodeData } from "~/components/code-demo";
+
+const code = `
+"use client";
+
+import { PhoneInput } from "flowbite-react";
+
+function Component() {
+    return (
```

Comment:
needs formatting, this looks like it has a 4 space indent instead of 2 which is project default config